library/include/rocblas-types.h

```diff
@@ -106,6 +106,13 @@ typedef enum rocblas_pointer_mode_ {
     rocblas_pointer_mode_device = 1
 } rocblas_pointer_mode;
 
+/*! \brief Indicates if layer is active*/
+typedef enum rocblas_layer_mode {
```

Comment:
Make the intention of these to be bitmasks with the 0x prefix, i.e.  0x1, 0x2, 0x4

Reply:
fixed: use binary literal from C++ 14 and available in clang and gcc since 2013